docs/load.rst

```diff
@@ -327,6 +327,78 @@ from the tables.
     db["emotion"].get()
 
 
+.. _streaming:
+
+Streaming
+---------
+
+:func:`audb.stream` provides a pseudo-streaming mode,
+which helps to load large datasets.
+It will only load ``batch_size`` number of rows
+from a selected table into memory,
+and downloads only matching media files
```

Comment:
The first part of the sentence is in future tense. therefore I think `download` is correct. 

```suggestion
and download only matching media files
```